package.json

```diff
@@ -45,7 +45,9 @@
     "test": "karma start ./karma.conf.js --single-run",
     "test:travis": "karma start ./karma.conf.js --single-run",
     "test:dev": "karma start ./karma.conf.js --no-single-run --auto-watch",
-    "test:cov": "karma start ./karma.conf.js --single-run --reporters coverage"
+    "test:cov": "karma start ./karma.conf.js --single-run --reporters coverage",
+    "pregh-pages": "git fetch origin",
```

Comment:
Right, ok. It's just more idiomatic so I usually put cleaning stuff into `pre-*`.